python/graphscope/dataset/p2p_network.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright 2021 Alibaba Group Holding Limited. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+
+from graphscope.client.session import get_default_session
+from graphscope.dataset.io_utils import DATA_SITE
+from graphscope.dataset.io_utils import download_file
+from graphscope.framework.graph import Graph
+
+
+def load_p2p_network(sess=None, prefix=None, directed=False):
```

Comment:
Also upload the uncompressed version (i.e. the folder named `p2p-network`) to the bucket, to coordinate with the k8s loading dataset utility.